src/System.Collections.Immutable/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard;
```

Comment:
The point is that a netstandard config can live in the netcoreapp package, because netcoreapp package will contain the netstandard shims. This will help with making some implementations reusable.